tests/e2e/performance/test_pvc_attachtime.py

```diff
@@ -157,26 +194,47 @@ def test_pod_start_time(self, pod):
         self.full_results = self.init_full_results(
             ResultsAnalyse(self.uuid, self.crd_data, self.full_log_path)
         )
+        self.full_results.add_key("storageclass", self.sc)
 
-        start_time = start_time_dict["web-server"]
-        logging.info(f"pod start time: {start_time} seconds")
-        if start_time > 30:
-            raise ex.PerformanceException(
-                f"pod start time is {start_time}," f"which is greater than 30 seconds"
+        time_measures = [t["web-server"] for t in start_time_dict_list]
```

Comment:
I wonder why the key is `web-server`. it impedes the readability of the code. we may need to change it maybe in a subsequent PR. 
@Avilir @ypersky1980

Reply:
@keesturam for that we need to change the pod image, and yes this can be done in subsequent PR.